### PR TITLE
Start using JS string template in JS code

### PR DESCRIPTION
**Pull request by sbc100**
It looks like these have been implements for a very long time in all the engines: https://caniuse.com/template-literals

I just chose of libraries to start with, and will continue with others if there are no objections.

### CODE REVIEW

**Comment by sbc100** [comment]
@brendandahl @kripken @juj what do you think about this direction?  

I think code-size-wise and performance-wise its mostly a wash, but maybe its win for maintainability/readability?

**Comment by sbc100** [comment]
My motivation is partly that contributors tend to assume they can use this feature and we should probably be consistent about using it vs not using it.

**Review by kripken — APPROVED** [review]
It is more readable I think, seems worthwhile.

File: src/library.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ mergeInto(LibraryManager.library, {
 #if ASSERTIONS
     // if exit() was called explicitly, warn the user if the runtime isn't actually being shut down
     if (keepRuntimeAlive() && !implicit) {
-      var msg = 'program exited (with status: ' + status + '), but keepRuntimeAlive() is set (counter=' + runtimeKeepaliveCounter + ') due to an async operation, so halting execution but not exiting the runtime or preventing further async execution (you can use emscripten_force_exit, if you want to force a true shutdown)';
+      var msg = `programprogram exited (with status: ${status}), but keepRuntimeAlive() is set (counter=${runtimeKeepaliveCounter}) due to an async operation, so halting execution but not exiting the runtime or preventing further async execution (you can use emscripten_force_exit, if you want to force a true shutdown)`;
```
Comment (curiousdannii) [inline src/library.js:95]: "programprogram"

**Comment by juj** [comment]
This looks perfect!

**Pull request merged**